cmd/migration-assist/commands/postgres.go

```diff
@@ -167,6 +168,13 @@ func runPostMigrateCmdF(c *cobra.Command, args []string) error {
 
 	err = postgresDB.RunEmbeddedMigrations(queries.Assets(), "post-migrate", baseLogger)
 	if err != nil {
+		if strings.Contains(err.Error(), "pq: string is too long for tsvector") {
```

Comment:
Btw, I didn't give enough time on this. But this can be improved by looking at the error code:

```go
var pqErr *pq.Error
case errors.As(err, &pqErr):
		if pqErr.Code == "54000" {
			return true
		}
```

Ref: 
1. https://github.com/lib/pq/blob/3d613208bca2e74f2a20e04126ed30bcb5c4cc27/error.go#L293C3-L293C8
2. https://github.com/postgres/postgres/blob/ab61c40bfa2ba1887fee304b2ef5306a14a7248c/src/backend/utils/adt/tsvector.c#L222

Reply:
I'll create a separate PR, thanks!